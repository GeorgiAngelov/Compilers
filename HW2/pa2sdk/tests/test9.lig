function f() {
      return (g(g(x), g(y), 1+3));
}
