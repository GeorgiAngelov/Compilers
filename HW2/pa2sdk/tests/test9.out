Genuine Liger: yes
Function g:
   Arity mismatch!
   Defined: no
   Calls: 3
Function f:
   Arity: 0
   Defined: yes
   Calls: 0
